src/micromamba/NOTES.md

```diff
@@ -0,0 +1,20 @@
+## Version number specification
+
+Soft version matching is *not* supported, meaning that `"1"` and `"1.0"` are not
+valid values of the `version` parameter. The full version number must be specified
+like `"1.0.0"`.
+
+## Channels
+
+By default, `micromamba` configures no channels. If you would like to set `conda-forge`
+as a default channel, then use
+
+```json
+"features": {
+  "ghcr.io/eitsupi/mamba-devcontainer-features/micromamba:latest": {
```

Comment:
Given the possibility of breaking changes change the type from string to array in the future (devcontainers/spec#57), it is better to specify `0` than `latest` here, I think.

```suggestion
  "ghcr.io/eitsupi/mamba-devcontainer-features/micromamba:0": {
```

Reply:
You are right. Let's update and merge this!

Reply:
I don't like pre-release semver because it is complicated and confusing. According to my understanding, pre-release, breaking changes are indicated by `0.(X+1)`.

Several people (including me) prefer to skip pre-release versioning and go directly to `1.0.0` where the usual rules apply. In that case, we could use `1` here in anticipation of the release.

Reply:
Yes, in fact I believe the various Dev Container Features often start with version `1.0.0`.
I tend to start with `0.1.0` as an excuse for breaking changes, but you are welcome to version this as you see fit.

Reply:
But I am also fine with `0.1`.